unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ParcelScene.cs

```diff
@@ -684,6 +684,32 @@ private void RemoveEntityComponent(IDCLEntity entity, string componentName)
                         }
                     }
                     return;
+                default:
```

Comment:
With this new `default` logic, do the `OnClick.NAME`, `OnPointerDown.NAME`, `OnPointerUp.NAME`, `OnPointerHoverEnter.NAME`, `OnPointerHoverExit.NAME` cases of that `switch` still make sense? couldn't those work fine with the new `default` as well?

Reply:
yes and no :P
having those as switch cases will make it faster than the default case which iterate through the component list to find the component to remove.
Ideally, for better performance, we should add a case in that switch for every component and the default case is mostly to handle the components that we forgot to add there